syntax = "proto3";
package rdma;

message RDMAConnResponseMgmt {
    uint64 buffer = 1;
    uint32 rkey = 2;
    uint32 qp_num = 3;
    uint32 lid = 4;
    repeated uint32 gid = 5 [packed=true];
    uint32 psn = 6;
}
